### ISSUE TITLE
Update Discord experimental Docker image

### ISSUE THREAD

**Issue by Beanow** [sourcecred/sourcecred#1737]
MetaGame uses this tag to experiment with both initiatives and Discord.
Current image has Discord, but not Initiatives.

Would like to rebase and update the image.
cc @hammadj

**Comment by Beanow** [comment]
The branch which had the Discord prototype is updated.
https://github.com/sourcecred/sourcecred/tree/discord

I've done a quick test of loading some Initiative files plus a Discord server and seems to work OK.

I've tagged the commits on GitHub:
https://github.com/sourcecred/sourcecred/tree/wip-discord-1
https://github.com/sourcecred/sourcecred/tree/wip-discord-2

Correspondingly I've updated the [DockerHub tags](https://hub.docker.com/r/sourcecred/sourcecred/tags):
- `wip-discord` is updated to be `wip-discord-2`
- `wip-discord-1` is the previous version
- `wip-discord-2` is the updated version

I believe that the GH action will rebuild it's docker image every run based on the `wip-discord` tag.
So this should automatically start working.
